UDV-Core/src/Modules/ConsultDoc/DocumentBrowser.js

```diff
@@ -227,7 +230,8 @@ export function DocumentBrowser(browserContainer, documentController) {
         document.getElementById('docBrowserInfo').innerHTML =
                             "No document to display according to your research";
         document.getElementById('docBrowserIndex').innerHTML = "No doc";
-
+        $('#docDeleteButton').hide();
+        $('#docUpdateButton').hide();
```

Comment:
Same remark: why DocumentBrowser knows about Delete and update buttons ?

Reply:
same remark